docs/release-notes/next/PT-166114135_fine_tune_channels_for_fortuna.md

```diff
@@ -0,0 +1,5 @@
+* Disables usage of provided StateChannels WebSocket API for generalized
+  accounts. This is done because FSM is not fully aware of generalized
+  accounts. It is highly recommended for state channels' users **not to
```

Comment:
Recommended note also in:
* User doc re usage of state channels - I expect https://github.com/aeternity/protocol/blob/master/node/api/channels_api_usage.md
* Common symptoms/resolutions if any in https://github.com/aeternity/aeternity/wiki/Troubleshooting